gcovr/configuration.py

```diff
@@ -763,6 +763,13 @@ def merge_options_and_set_defaults(
         help="Report the decision coverage. For HTML and JSON report.",
         action="store_true",
     ),
+    GcovrConfigOption(
+        "show_calls",
+        ["--calls"],
+        group="output_options",
+        help="Report the calls coverage. For HTML report.",
+        action="store_true",
+    ),
```

Comment:
Would `--html-calls` fit better? It only affects the HTML report and the coverage is always read from gcov output.

Reply:
When I think about it more closely, the JSON format should also be expanded to include the calls. Otherwise no HTML can be created with the calls from merged JSON files.

Reply:
Yes, thank you so much. I plan to add support in the JSON format as it is the intermediate format to merge reports in gcovr (isn't it?) and it is a key feature, but as for this pr --html-calls is clearer

Reply:
Please keep the name and add the information to the JSON report with this PR.

Reply:
Added the support to merge call coverage and load from .json

Changed also the test case to test call merge (make merge.html)